erizo_controller/erizoClient/src/Erizo.js

```diff
@@ -3,14 +3,20 @@ import { LicodeEvent, RoomEvent, StreamEvent } from './Events';
 import Stream from './Stream';
 import Logger from './utils/Logger';
 
-// eslint-disable-next-line 
-require('expose-loader?adapter!../lib/adapter.js');
-// eslint-disable-next-line
-require('script-loader!./utils/L.Resizer.js');
+// #if process.env.INCLUDE_ADAPTER == 'TRUE'
+import adapter from 'webrtc-adapter'; // eslint-disable-line
+// #endif
+
+require('script-loader!./utils/L.Resizer.js');  // eslint-disable-line
 
 
 const Erizo = {
+  // #if process.env.INCLUDE_SOCKETIO == 'TRUE'
   Room: Room.bind(null, undefined, undefined, undefined),
+  // #endif
+  // #if process.env.INCLUDE_SOCKETIO != 'TRUE'
```

Comment:
I assume `#else` would work here. Consider using it

Reply:
I thought so too, but nope :/